diff --git a/BUILD.gn b/BUILD.gn
index 62210c2f6..053fbff6d 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -228,7 +228,7 @@ group("pdfium_public_headers") {
   ]
 }
 
-component("pdfium") {
+shared_library("pdfium") {
   output_name = "embedpdf"
   libs = []
   configs += [ ":pdfium_strict_config" ]
@@ -263,6 +263,9 @@ component("pdfium") {
       "gdi32.lib",
       "user32.lib",
     ]
+    sources = [
+      "resources.rc",
+    ]
   }
 
   if (is_mac) {
