diff --git a/BUILD.gn b/BUILD.gn
index 908763359..8c4db2aa6 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -225,12 +225,7 @@ group("pdfium_public_headers") {
   ]
 }
 
-pdfium_component_target_type = "component"
-if (pdf_is_complete_lib) {
-  pdfium_component_target_type = "static_library"
-}
-
-target(pdfium_component_target_type, "pdfium") {
+shared_library("pdfium") {
   output_name = "pdfium"
   libs = []
   configs += [ ":pdfium_strict_config" ]
@@ -268,6 +263,9 @@ target(pdfium_component_target_type, "pdfium") {
       "gdi32.lib",
       "user32.lib",
     ]
+    sources = [
+      "resources.rc",
+    ]
   }
 
   if (is_mac) {
